index.html

```diff
@@ -14,73 +14,67 @@
-                </div>
-              </div>
-        </section>
-
-
-                 <div class="entire-form-content">
-                     <div class="hero-body form-title">
-                         <p>Explore a place in time or space</p>
-                     </div>
-                     <div>
-                         <form>
-                             <div class="search-form">
-                                 <div class="search-form-item">
-                                     <label for="form-destination">Destination:</label><br>
-                                     <input type="text" id="form-destination" name="form-destination">
-                                 </div>
-                                 <div class="search-form-item">
-                                     <label for="form-time">Time Period:</label><br>
-                                     <input type="text" id="form-time" name="time-period">
-                                 </div>
-                                 <div class="search-form-item">
-                                     <label for="form-time">Travel Dates:</label><br>
-                                     <input type="date" id="form-dates" name="travel-dates">
-                                 </div>
-                             </div>
-                         </form>
-                          <div class="search-form-button">
-                              <button>Begin My Journey
-                              </button>
-                          </div>
-                     </div>
-                 </div>
-       </header>
+   <header>
+       <!-- Header -->
+       <section>
+           <div class="nav-container">
+
+               <a href="./"><img src="./assets/img/logos/logo-icon/logo-icon-white.svg" alt="Who Travels Logo" class="logo"></a>
+               <nav class="nav-menu">
+                   <ul class="menu">
+                       <li class="nav-li"><a href="../book/" class="top-nav" id="book">Book Trip</a></li>
+                       <li class="nav-li"><a href="../where-to/" class="top-nav" id="destinations">Destinations</a></li>
+                       <li class="nav-li"><a href="../about/" class="top-nav" id="about">About Us</a></li>
+                       <li class="nav-li"><a href="../contact/" class="top-nav" id="contact">Contact</a></li>
+
+                   </ul>
+                   <div class="hamburger-icon">
+                       <span class="hamburger"></span>
+                       <span class="hamburger"></span>
+                       <span class="hamburger"></span>
+                   </div>
+               </nav>
+           </div>
+       </section>
+       <section class="hero">
+           <div class="hero-img-box">
+               <img class="hero-img" src="./assets/img/heros/hero-home.jpg" alt="photo of man looking at stars">
+           </div>
+           <div class="hero-content">
```

Comment:
Was the code copied/pasted from another version? The adjustment for excluding the hero-h1 from screenreading are gone. Easy enough to adjust once this is merged though.